src/cdk-experimental/combobox/combobox-panel.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+
+@Directive({
+  selector: 'ng-template[cdkComboboxPanel]',
+  exportAs: 'cdkComboboxPanel',
+  host: {
+    'role': 'combobox'
+  }
+})
+export class CdkComboboxPanel<V = unknown> {
```

Comment:
I'd probably prefer to use `T` as the template type here (and elsewhere)

Reply:
Noted.